programs/test/selftest.c

```diff
@@ -419,4 +479,3 @@ int main( int argc, char *argv[] )
     /* return() is here to prevent compiler warnings */
     return( MBEDTLS_EXIT_SUCCESS );
 }
-
```

Comment:
In theory, the empty new line at the end of a non-empty C file is mandated by the standard.